src/services/tweets.service.ts

```diff
@@ -42,7 +53,21 @@ export const updateTweet = (
     updatedAt: string;
   }
 ) => {
-  return updateOne(id, newData);
+  const tweetOldData = await getOneById(id);
+  const updateResult = await updateOne(id, newData);
+
+  if (updateResult) {
```

Comment:
what do you check here?
if error we did not come here
if object we had always true